test/rules/std_fix_auto_test.py

```diff
@@ -1,6 +1,6 @@
 """Automated tests for fixing violations.
 
-Any files in the /tests/fixtures/linter/autofix directoy will be picked up
+Any files in the test/fixtures/linter/autofix directory will be picked up
```

Comment:
A separate PR about this little fix, or maybe not?

Reply:
Well, I suppose this PR will be squashed so the commit message / git blame would be misleading for those. There were actually 2 cases like that so I created a separate PR after all https://github.com/sqlfluff/sqlfluff/pull/2179